packages/tools/pie-wrappers-react/scripts/react-gen-wrapper.js

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs-extra';
+import { appendFile } from 'fs'
+
+// fetches custom-elements.json file from the root
+const loadJSON = (path) => JSON.parse(fs.readFileSync(new URL(path, import.meta.url)));
+
+const customElementsObject = loadJSON(`../custom-elements.json`);
+
+export let components = []
+
+const customElements = Object.entries(customElementsObject)
+
+// sort through customElements array and put all components into a separate array
+customElements.forEach(([key, value]) => {
+    if ( key === 'modules') {
+        return value.forEach(k => {
+            k.declarations.forEach((decl) => {
+                decl.customElement === true ? components.push(decl) : ''
+            })
+        })
+    }
+})
+
+/**
+ * function to fetch events from the component
+ *
+ * @param {*} component object from within components array
+ * @param {*} event empty array to be populated with events
+ */
+function getEvents(component, events) {
+       if (component?.events) {
+            events.push(
+                component?.events
+                    .filter((event) => !!event.name)
+                    .map((event) => {
+                        return {
+                            name: event.name,
+                            type: event.type?.text || 'Event',
+                            description: event.description,
+                        };
+                    })
+            );
+        }
+    }
+
+// create wrapper src code and add to index.ts file
+components.forEach(component => {
+    const events = []
+    getEvents(component, events)
+
+    const componentSrc = `import * as React from 'react';
```

Comment:
Can you add a couple of line breaks before the `import`?

The temporary code looks a bit squeezed.

```
...
declare global {
    interface HTMLElementTagNameMap {
        'pie-button': PieButton;
    }
}import * as React from 'react';
...
```